.github/workflows/ensure_label.yml

```
@@ -0,0 +1,20 @@
+name: PR should have a language label
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  label:
+    runs-on: ubuntu-20.04
```

Most of our workflows use ubuntu-latest and I think this one would benefit from it.
```suggestion
    runs-on: ubuntu-latest
```